src/desktop.md

```diff
@@ -315,6 +315,38 @@ from a Flutter desktop app, see the [Desktop Photo Search sample][].
 [msix package]: {{site.pub}}/packages/msix
 [Desktop Photo Search sample]: {{site.github}}/flutter/samples/tree/master/experimental/desktop_photo_search
 
+###### Create a .pfx certificate.
+
+To publish on the Windows Store or Direct download with the help of MSIX Installer, you need to give your app a digital signature in form of a .pfx certificate. Use the following instructions to generate a self-signed .pfx certificate.
+
+* You would need to download [OpenSSL][] to generate your certificates.
+* Go to where you installed the OpenSSL. It can be:
+    * C:\Program Files\OpenSSL-Win64\bin
+* Run cmd from this folder, that it looks like this in cmd.
+    * C:\Program Files\OpenSSL-Win64\bin>
+      or
+    * make “C:\Program Files\OpenSSL-Win64\bin” a environment path variable to access “OpenSSL” anywhere.
+* Now, use the following command:
+    * Generate a private key.
```

Comment:
```suggestion
1. Go to where you installed OpenSSL, for example:
   `C:\Program Files\OpenSSL-Win64\bin`.
1. Set an environment variable so that you can access `OpenSSL` from anywhere:<br>
   `make "C:\Program Files\OpenSSL-Win64\bin"`
1.  Generate a private key as follows:<br>
```